data_labeling_examples/bulk_labeling_python/README.MD

```diff
@@ -0,0 +1,103 @@
+    Label Set: cat , dog 
+    FIRST_MATCH_REGEX_PATTERN : ^([^\/]*)\/.*$
+    Result of FIRST_REGEX_MATCH labeling algorithm: 
+    PetCat1.jpeg will be labeled with cat label
+    PetCat2.jpeg will be labeled with cat label
+    PetDog1.png will be labeled with dog label
+    PetDog2.png will be labeled with dog label
+    ```
+
+
+    - CUSTOM_LABELS_MATCH: This algorithm takes object storage path as input along with the label that needs to be applied to records under that path. Only root level path is supported. Multiple labels can also be assigned to a given path. The labeling algorithm for this case is .
+
+    ```
+    Consider a dataset having following records:
+    cat/cat1.jpeg, cat/cat2.jpeg, dog/dog1.jpeg, dog/dog2.jpeg
+    Labels in dataset: dog, pup, cat, kitten
+    CUSTOM_LABELS={ "dog/": ["dog","pup"], "cat/": ["cat", "kitten"] }
+    Result of CUSTOM_LABELS_MATCH algorithm: 
+        cat/cat1.jpeg will be labeled with cat and kitten labels
+        cat/cat2.jpeg will be labeled with cat and kitten labels
+        dog/dog1.png will be labeled with dog and pup labels
+        dog/dog2.png will be labeled with dog and pup labels
+    ```
+
+### Requirements
+1. An Oracle Cloud Infrastructure account. <br/>
+2. A user created in that account, in a group with a policy that grants the desired permissions. This can be a user for yourself, or another person/system that needs to call the API. <br/>
+3. A key pair used for signing API requests, with the public key uploaded to Oracle. Only the user calling the API should be in possession of the private key. For more information, see [Configuring the SDK](https://docs.oracle.com/en-us/iaas/tools/python/2.62.0/installation.html#install). <br/> 
+4. Python 3.0.0+ <br/>
+
+For more information [SDK for Python](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/pythonsdk.htm)
+
+### Configurations
+
+Add the following configurations in config.properties file in the project to run the scripts:
+
+```
+import multiprocessing as mp
+# for help, run:
+# python3 help.py
+
+# config file path
+CONFIG_FILE_PATH = "~/.oci/config"
+
+# config file profile
+CONFIG_PROFILE = "DEFAULT"
+
+# region identifier of DLS Dataset
+REGION_IDENTIFIER = "uk-london-1"
+
+# service_endpoint
+SERVICE_ENDPOINT_DP = f"https://dlstest-dp.{REGION_IDENTIFIER}.oci.oraclecloud.com"
+
+# compartment where DLS Dataset exists
+COMPARTMENT_ID = "ocid1.compartment.oc1..aaaaaaaamrnszmfhl7xzawtef3rl6gmsb62tcoag32yiv5ujorognkuzrqfq"
+
+# ocid of the DLS Dataset
+DATASET_ID = "ocid1.datalabelingdatasetint.oc1.uk-london-1.amaaaaaaniob46iarr2zttq7c5th3jfqwab7d3vrq4daa52tcnnwhkgrowca"
+
+# an array where the elements are all of the labels that you will use to annotate records in your DLS Dataset with. Each element is a separate label.
+LABELS = ["dog", "cat"]
+
+# the algorithm that will be used to assign labels to DLS Dataset records
```

Comment:
Please mention possible values for labeling algorithms in this comment.

Reply:
Comment -> [" ENUMS  for labeling algorithm "FIRST_LETTER_MATCH", "FIRST_REGEX_MATCH",  "CUSTOM_LABELS_MATCH" ]

Reply:
I meant that you can provide the possible values as comment in the config's comment for easier resference by user.